api/src/membership/test/membership_test.py

```diff
@@ -0,0 +1,96 @@
+from dataclasses import dataclass
+from datetime import date, timedelta, timezone
+from datetime import datetime as dt
+from typing import Any, Dict, List, Optional, Set, Tuple, TypeVar
+
+import core
+import core.models
+import membership
+import membership.models
+import membership.views
+import messages
+import messages.models
+import shop
+import shop.models
+from membership.membership import get_membership_summaries
+from membership.models import Member, Span
+from test_aid.test_base import FlaskTestBase, ShopTestMixin
+
+
+class MembershipTest(ShopTestMixin, FlaskTestBase):
+    models = [membership.models, messages.models, shop.models, core.models]
+
+    def test_get_membership_summaries_old(self):
```

Comment:
Undescriptive test name.

The test name should say what is the test purpose. Like "test formatting of membership summary when lab access is expired"

This is applicable for the other tests as well.